cdbot/cogs/cyber.py

```diff
@@ -360,6 +360,74 @@ def __init__(self, bot: Bot):
         await ctx.send(f"There are {len(game_r.members)} that have completed Cyberstart Game. Out of them, "
                        f"{len(true_r.members)} have also completed Essentials and Assess.")
 
+    @command()
+    async def elitecountdown(self, ctx: Context, event_name: str = None, event_level: str = None):
+        """
+        Gets the time until elite, and shows in an embed
+        """
+        valid_elite_locations = ["London", "Birmingham", "Lancaster "]
+
+        # Validate and normalise inputs
+        event_name = event_name.capitalize()
+        if event_name:
+            if not (event_name in valid_elite_locations):
+                try:
+                    event_name = ELITE_ALISES[event_name]
+                except KeyError:
+                    content = f"{event_name} is not a valid elite location! " \
+                        "Please choose from 'London', 'Birmingham', 'Lancaster'"
```

Comment:
You could have non hard-coded event locations.

Reply:
Specifically, you already have a constant which stores all of the valid locations, so you could just do something like

```python
f"Please chose from {[f"'{location}'" for location in valid_elite_locations].join(", ")}"
```

Reply:
Do you mean
```python
f"Please chose from {", ".join(valid_elite_locations)}"
```

Reply:
Or whatever you kids do these days to format a string inside a string (`f"`, `%` or `.format()`)

Reply:
Sure, assuming you don't want single quotes around the items.